GCRCatalogs/register.py

```diff
@@ -55,7 +81,7 @@ def content(self):
             self._content = load_yaml_local(self.path)
         return self._content
 
-
+    
```

Comment:
```suggestion

```

Reply:
Oops!  I've eliminated the extra spaces.